local_dev/start_containerd.sh

```diff
@@ -2,55 +2,34 @@
 # abort if any of the following commands fails or variables are undefined
 set -eu
 
-KUBERNETES_VERSION="${KUBERNETES_VERSION:-v1.15.0}"
-CALICO_VERSION="${CALICO_VERSION:-v3.8}"
+KUBERNETES_VERSION="${KUBERNETES_VERSION:-stable}"
 
-# Setup minikube
+# Setup minikube, requires minikube >= 1.12.1
 minikube delete
 minikube config set embed-certs true
 minikube start \
     --memory 4096 \
-    --cpus 4 \
-    --network-plugin=cni \
+    --cpus 2 \
```

Comment:
Any reason to reduce the number of CPUs? IMHO this can be dropped since it's the default anyway.

Reply:
This was accidentally committed.